src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -6,6 +6,7 @@ import {
   DATA_TABLE_CLASSES,
   EXPANDED_CLASS,
   EXPAND_CLASS,
+  GENERIC_SIZES,
   GENERIC_SIZE_CLASSES,
```

Comment:
Aparently, classes.ts holds the same objects for the generic classes as `GENERIC_SIZES` & `GENERIC_SIZE_CLASSES`. We need remove `GENERIC_SIZES` and use `GENERIC_SIZE_CLASSES` in all the cases.